packages/projects-docs/pages/learn/boxy/overview.mdx

```diff
@@ -0,0 +1,60 @@
+---
+title: Meet Boxy
+description: Introduction to Boxy the AI-powered coding companion
+---
+
+import { Callout } from 'nextra-theme-docs'
+
+
+# Meet Boxy
+
+Boxy is an AI-powered chat tool that is built directly into your editor. Boxy is aware of your code and can answer questions and make suggestions that take into account best practices as well as conventions that are specific to your project.  
+
+## Boxy in the editor
+If you have granted AI permissions in the workspace settings. You will notice the newest addition to your team sitting next to your avatar. The easiest way to work with Boxy is to select a piece of code and a saved prompt.  
+
+The easiest way to work with Boxy is to select a piece of code and a saved prompt. When you select the code, you will see a toolbar hover above the topline of selected code. From here you can either open the devtool to chat with Boxy or you can run a saved prompt.
```

Comment:
I think this paragraph says the same thing above, but it's more descriptive. Maybe you forgot it here?